src/tests/unit/tests/DetailsView/components/load-assessment-button.test.tsx

```diff
@@ -11,33 +12,63 @@ import {
+import { mount, shallow } from 'enzyme';
+import { ActionButton } from 'office-ui-fabric-react';
 import * as React from 'react';
 
-import { Mock } from 'typemoq';
+import { IMock, It, Mock } from 'typemoq';
 
 describe('LoadAssessmentButton', () => {
-    const detailsViewActionMessageCreatorMock = Mock.ofType(DetailsViewActionMessageCreator);
-    const assessmentDataParserMock = Mock.ofType(AssessmentDataParser);
-    const urlParserMock = Mock.ofType(UrlParser);
-    const loadAssessmentHelperMock = Mock.ofType(LoadAssessmentHelper);
+    let detailsViewActionMessageCreatorMock: IMock<DetailsViewActionMessageCreator>;
+    let assessmentDataParserMock: IMock<AssessmentDataParser>;
+    let urlParserMock: IMock<UrlParser>;
+    let loadAssessmentHelperMock: IMock<LoadAssessmentHelper>;
+    let handleLoadAssessmentButtonClickMock: IMock<(event: React.MouseEvent<any>) => void>;
+    let event;
+    let props: LoadAssessmentButtonProps;
+    let deps: LoadAssessmentButtonDeps;
+    let tabStoreData: TabStoreData;
+    let assessmentStoreData: AssessmentStoreData;
 
-    const tabStoreData = {
-        id: 5,
-    } as TabStoreData;
+    beforeEach(() => {
+        event = {} as React.MouseEvent<any>;
+        detailsViewActionMessageCreatorMock = Mock.ofType(DetailsViewActionMessageCreator);
+        assessmentDataParserMock = Mock.ofType(AssessmentDataParser);
+        urlParserMock = Mock.ofType(UrlParser);
+        loadAssessmentHelperMock = Mock.ofType(LoadAssessmentHelper);
+        handleLoadAssessmentButtonClickMock = Mock.ofInstance(e => {});
 
-    const assessmentStoreData = {} as AssessmentStoreData;
+        props = {
+            deps,
+            tabStoreData,
+            assessmentStoreData,
+            handleLoadAssessmentButtonClick: handleLoadAssessmentButtonClickMock.object,
+        } as LoadAssessmentButtonProps;
 
-    const deps = {
-        detailsViewActionMessageCreator: detailsViewActionMessageCreatorMock.object,
-        assessmentDataParser: assessmentDataParserMock.object,
-        urlParser: urlParserMock.object,
-        loadAssessmentHelper: loadAssessmentHelperMock.object,
-    } as LoadAssessmentButtonDeps;
-    const props = { deps, tabStoreData, assessmentStoreData } as LoadAssessmentButtonProps;
+        deps = {
+            detailsViewActionMessageCreator: detailsViewActionMessageCreatorMock.object,
+            assessmentDataParser: assessmentDataParserMock.object,
+            urlParser: urlParserMock.object,
+            loadAssessmentHelper: loadAssessmentHelperMock.object,
+        } as LoadAssessmentButtonDeps;
+
+        tabStoreData = {
+            id: 5,
+        } as TabStoreData;
+
+        assessmentStoreData = {} as AssessmentStoreData;
+    });
 
     it('should render per the snapshot', () => {
         const rendered = shallow(<LoadAssessmentButton {...props} />);
-        expect(rendered.getElement()).toMatchSnapshot();
+        expect(rendered).toMatchSnapshot();
```

Comment:
Think you're missing `.getElement()` here because the snapshot is empty.

Reply:
good catch!